README.md

```diff
@@ -2,14 +2,49 @@
 
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
+  1. One algorithm may appear to perform better than another asymptotically,
+     but if the worst case data set for the algorithm with a better average case 
+     ends up being a very common data set for the problem at hand, then you'll
+     want to choose the algorithm that has an asymptotically worse average case,
+     but a less likely and/or less detrimental worst case.
+  2. Some algorithms (like insertion sort) work better than more complicated
+     algorithms (like quicksort) when applied to very small data sets. Terms
+     that are asymptotically insignificant on large data sets can be much more
+     significant on small data sets.
```

Comment:
Not sure if I'm following you here -- what terms are more significant for large input sizes?